From ac782ed2990d5d8149c6796aa9aaf03044ad1446 Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Wed, 10 Feb 2016 13:45:50 -0800
Subject: [PATCH 85/86] Ticket #48536 - Crash in slapi_get_object_extension

Description: commit 091a5f5daf3fa378f029e293c5358ae9be9f548e introduced
a compier warning: format '%x' expects argument of type 'unsigned int',
but argument 4 has type 'Slapi_PBlock * {aka struct slapi_pblock *}

Replacing '%x' with '%p'.

(cherry picked from commit b31749f2a0bed838712719b2e6533bed0a4144e0)
(cherry picked from commit 7fb593fe56dbbce06f707bd9807bf14748273ea2)
---
 ldap/servers/plugins/acl/acl.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ldap/servers/plugins/acl/acl.c b/ldap/servers/plugins/acl/acl.c
index d56bed6..7b7fea1 100644
--- a/ldap/servers/plugins/acl/acl.c
+++ b/ldap/servers/plugins/acl/acl.c
@@ -319,7 +319,7 @@ acl_access_allowed(
 
 	if (pb != aclpb->aclpb_pblock) {
 		slapi_log_error(SLAPI_LOG_FATAL, plugin_name,
-		                "acl_access_allowed: Resetting aclpb_pblock 0x%x to pblock addr 0x%x\n",
+		                "acl_access_allowed: Resetting aclpb_pblock %p to pblock addr %p\n",
 		                aclpb->aclpb_pblock, pb);
 		aclpb->aclpb_pblock = pb;
 	}
@@ -942,7 +942,7 @@ acl_read_access_allowed_on_entry (
 	 
 	if (pb != aclpb->aclpb_pblock) {
 		slapi_log_error(SLAPI_LOG_ACL, plugin_name,
-		                "acl_read_access_allowed_on_entry: Resetting aclpb_pblock 0x%x to pblock addr 0x%x\n",
+		                "acl_read_access_allowed_on_entry: Resetting aclpb_pblock %p to pblock addr %p\n",
 		                aclpb->aclpb_pblock, pb);
 		aclpb->aclpb_pblock = pb;
 	}
-- 
2.4.3

